assets-and-images.md

```diff
@@ -248,6 +248,50 @@ flutter:
 
 The `lib/` is implied, so it should not be included in the asset path.
 
+## Sharing assets with the underlying platform
+
+It is possible in the underlying platform to access assets included in Flutter. 
```

Comment:
I find that sentence a bit hard to read. What about "Flutter assets are readily available to platform code via AssetManager on Android and NSBundle on iOS."